command/genesis/utils.go

```diff
@@ -164,6 +164,41 @@ func parseBurnContractInfo(burnContractInfoRaw string) (*polyCommon.BurnContract
 	}, nil
 }
 
+type baseFeeInfo struct {
+	baseFee   uint64
+	baseFeeEM uint64
+}
+
+// parseBaseFeeConfig parses provided base fee configuration and returns baseFeeInfo
+func parseBaseFeeConfig(baseFeeConfigRaw string) (*baseFeeInfo, error) {
+	baseFeeInfo := &baseFeeInfo{command.DefaultGenesisBaseFee, command.DefaultGenesisBaseFeeEM}
+
+	baseFeeConfig := strings.Split(baseFeeConfigRaw, ":")
+	if len(baseFeeConfig) > 2 {
```

Comment:
I think we should check if there is more than one `:` (two values) in validate

Reply:
And ` &baseFeeInfo{command.DefaultGenesisBaseFee, command.DefaultGenesisBaseFeeEM}` we should return only if flag is not specified